feature-configs/deploy/gatekeeper/mutations.yaml

```diff
@@ -0,0 +1,1446 @@
+---
```

Comment:
please split this file into
1. CRD
2. Namespace
3. RBAC (roles and service accounts)
4. deployment + service
5. webhook